### PR TITLE
Sync

### PR DESCRIPTION
**Pull request by Tjoosten**
## Misc. changes 

- Synched the branch. 
- Reverted 2 totally wrong commits on my branch.
- Added `bug`, `license` and `repository` attribute to `package.json`.
  - That also fixed the warning that was throwed by `npm install`.
- Clean up the use statements.
- Improved several docblocks.
- Added laravel debugbar.

### CODE REVIEW

File: app/Http/Controllers/StationController.php [hunk 1]
```diff
@@ -9,6 +9,8 @@
 use Illuminate\Support\Facades\App;
 use irail\stations\Stations;
 use ML\JsonLD\JsonLD;
+USE Negotiation\FormatNegotiator;
```
Comment (pietercolpaert) [inline app/Http/Controllers/StationController.php]: Why capitals here?

**Comment by pietercolpaert** [comment]
Any idea why the travis-ci doesn't work?

**Comment by Tjoosten** [comment]
Not now. I'm investigating this

**Comment by Tjoosten** [comment]
@pietercolpaert TravisCi is now working correctly.

File: app/Http/Controllers/ClassicRedirectController.php [hunk 2]
```diff
@@ -45,8 +46,9 @@ public function redirectBoardSingleStation($station_provided_string)
      * Redirect classic liveboards URL to the new location
      * and interpret two station names
      *
-     * @param $station
-     * @param $station2
+     * @param  integer $station  The first station.
+     * @param  integer $station2 The second station.
```
Comment (pietercolpaert) [inline app/Http/Controllers/ClassicRedirectController.php]: Integer? Should be a string, right?

**Comment by pietercolpaert** [comment]
Before merging:
 * 1 file contains USE in capitals (see in line comments)
 * Doc block says stationTo is an integer, but it's a string

**Comment by Tjoosten** [comment]
@pietercolpaert This is now fixed with the fix commit

**Pull request merged**